### DISCUSSION TITLE
Rack diagrams not rendering properly

### DISCUSSION THREAD

**Discussion by Almeus** [netbox-community/netbox#9171] [Q&A] [answered]
Initially the rack diagrams shown when you look at a rack's info rendered normally, but when I started adding devices they started showing all scrunched up with a few letters of each device name overlapping the U number. I've tried different zoom levels and other browsers and they all show like this. 
What could I have changed to get them like [this?[](url)](![rack_issue](https://user-images.githubusercontent.com/48161435/164018869-6859e5ee-d4c5-47ad-af73-9874c9d61076.PNG))

**Comment by candlerb** [comment, marked as answer]
What version of Netbox are you running? Have you recently upgraded?

Try a shift-reload to flush your browser cache.

**Reply by Almeus** [reply]
I am running version 3.1.6. Clearing the cache didn't help. I'm using Chrome, but I've looked on Firefox as well with the same result.

**Reply by candlerb** [reply]
Have a look in your browser console then (developer tools), see if any error messages are logged regarding permissions or non-existent files.

I repeat the question: is this a fresh installation, or have you upgraded from an older version?

**Reply by candlerb** [reply]
Also, have you changed these settings, either in `configuration.py` or in Admin > Config Revisions?

```
RACK_ELEVATION_DEFAULT_UNIT_HEIGHT
RACK_ELEVATION_DEFAULT_UNIT_WIDTH
```

The defaults (when unset) are 22 and 220 respectively.

**Reply by Almeus** [reply]
I have not upgraded from a prior version

**Reply by Almeus** [reply]
RACK_ELEVATION_DEFAULT_UNIT_WIDTH was set to 0. Restoring it to 220 fixed the issue
